constants/src/crypto.rs

```diff
@@ -0,0 +1,184 @@
+//! Crypto related.
+
+/// This is a lookup table containing the zero commiment of known Pre-RCT value
+/// and is used to avoid computation of the commitment.
+/// 
+/// Defined at:
+/// - <https://github.com/monero-project/monero/blob/893916ad091a92e765ce3241b94e706ad012b62a/src/ringct/rctOps.cpp#L44>
+#[rustfmt::skip]
+pub static ZERO_COMMITMENT_LOOKUP_TABLE: [(u64, [u8;32]); 173] = 
+[
+  (0_u64,                       [0x58, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66, 0x66] ),
```

Comment:
Can we calcualte these at runtime using a LazyLock, like I did here: https://github.com/Cuprate/cuprate/blob/1eaf32bee956b66591f1537201c16969d5ec8696/helper/src/commitment.rs#L55-L62